redis/app-redis-separate-pods.yaml

```diff
@@ -0,0 +1,136 @@
+# The following variables are derived as follows by the pipeline. DO NOT CHANGE:
+#
+#   - NAMESPACE:    product
+#   - SERVICE_NAME: product-component-branch
+#   - IMAGE_NAME:   registry-name/hmcts/product-component:branch
+#
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${NAMESPACE}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ${SERVICE_NAME}
+  namespace: ${NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ${SERVICE_NAME}
+  template:
+    metadata:
+      labels:
+        app: ${SERVICE_NAME}
+    spec:
+      dnsConfig:
+        options:
+        - name: single-request-reopen
+      containers:
+      - image: ${IMAGE_NAME}
+        name: ${SERVICE_NAME}
+        env:
+        - name: REDIS_HOST
+          value: "redis://${SERVICE_NAME}-redis:6379"
+        # These may be adjusted, but be mindful please
+        resources:
+          requests:
+            memory: "64Mi"
+            cpu: "200m"
+          limits:
+            memory: "256Mi"
+            cpu: "500m"
+
```

Comment:
Just out of curiousity are these resource limits accurate for Redis or just copy/pasted?

Reply:
These were largely taken from https://kubernetes.io/docs/tutorials/stateless-application/guestbook/#start-up-the-redis-master

Reply:
👍